kinto/core/resource/__init__.py

```diff
@@ -932,6 +932,9 @@ def _extract_filters(self):
                 'description': 'Invalid value for {}'.format(param)
             }
 
+            if value == '':
```

Comment:
I guess this test is not only for `_since` and for other fields it might be useful to be able to search for an empty value.

Reply:
I agree.

There is already a `if param == '_since'` a little below BTW

Reply:
OK. I'll change it to allow other fields.

BTW, `_before`, `last_modified`, `gt_last_modified` and `lt_last_modified` have same issue：

```
$ http https://kinto.dev.mozaws.net/v1/buckets/default/collections/tasks/records?_before= --auth 'token:my-secret'
HTTP/1.1 503 Service Unavailable
Access-Control-Expose-Headers: Backoff,Retry-After,Alert,Content-Length
Connection: keep-alive
Content-Length: 151
Content-Type: application/json
Date: Mon, 05 Feb 2018 15:12:32 GMT
Retry-After: 3
Server: nginx
X-Content-Type-Options: nosniff

{
    "code": 503,
    "errno": 201,
    "error": "Service Unavailable",
    "message": "Service temporary unavailable due to overloading or maintenance, please retry later."
}

$ http https://kinto.dev.mozaws.net/v1/buckets/default/collections/tasks/records?_since= --auth 'token:my-secret'
HTTP/1.1 503 Service Unavailable
Access-Control-Expose-Headers: Backoff,Retry-After,Alert,Content-Length
Connection: keep-alive
Content-Length: 151
Content-Type: application/json
Date: Mon, 05 Feb 2018 15:12:43 GMT
Retry-After: 3
Server: nginx
X-Content-Type-Options: nosniff

{
    "code": 503,
    "errno": 201,
    "error": "Service Unavailable",
    "message": "Service temporary unavailable due to overloading or maintenance, please retry later."
}

$ http https://kinto.dev.mozaws.net/v1/buckets/default/collections/tasks/records?last_modified= --auth 'token:my-secret'
HTTP/1.1 503 Service Unavailable
Access-Control-Expose-Headers: Backoff,Retry-After,Alert,Content-Length
Connection: keep-alive
Content-Length: 151
Content-Type: application/json
Date: Mon, 05 Feb 2018 15:13:21 GMT
Retry-After: 3
Server: nginx
X-Content-Type-Options: nosniff

{
    "code": 503,
    "errno": 201,
    "error": "Service Unavailable",
    "message": "Service temporary unavailable due to overloading or maintenance, please retry later."
}

$ http https://kinto.dev.mozaws.net/v1/buckets/default/collections/tasks/records?gt_last_modified= --auth 'token:my-secret'
HTTP/1.1 503 Service Unavailable
Access-Control-Expose-Headers: Backoff,Retry-After,Alert,Content-Length
Connection: keep-alive
Content-Length: 151
Content-Type: application/json
Date: Mon, 05 Feb 2018 15:13:39 GMT
Retry-After: 3
Server: nginx
X-Content-Type-Options: nosniff

{
    "code": 503,
    "errno": 201,
    "error": "Service Unavailable",
    "message": "Service temporary unavailable due to overloading or maintenance, please retry later."
}

$ http https://kinto.dev.mozaws.net/v1/buckets/default/collections/tasks/records?lt_last_modified= --auth 'token:my-secret'
HTTP/1.1 503 Service Unavailable
Access-Control-Expose-Headers: Backoff,Retry-After,Alert,Content-Length
Connection: keep-alive
Content-Length: 151
Content-Type: application/json
Date: Mon, 05 Feb 2018 15:13:53 GMT
Retry-After: 3
Server: nginx
X-Content-Type-Options: nosniff

{
    "code": 503,
    "errno": 201,
    "error": "Service Unavailable",
    "message": "Service temporary unavailable due to overloading or maintenance, please retry later."
}
```

Reply:
Good catch!

Then you could do something like:
```py
for f in filters:
    if f.field == self.model.modified_field and f.value == '':
        raise...
```